lib/colors.ts

```diff
@@ -160,6 +162,26 @@ export function toPercent(value: number): string {
   return `${clean(100 * value)}%`
 }
 
+function isHexNotation(value: string): boolean {
+  return /^#?([\da-f]{3}|[\da-f]{4}|[\da-f]{6}|[\da-f]{8})$/i.test(value)
+}
+
+const OUTPUT_FORMAT_MAP: Partial<Record<Mode, OutputFormats>> = {
```

Comment:
Why do we need key→value map if they are equal?

What about:

```js
// stores/formats.js
export const OUTPUT_FORMATS = Object.keys(formats.get())

// lib/colors.js
function modeToFormat(mode: string) {
  if (mode in OUTPUT_FORMATS) …
}
```

Reply:
Added `OUTPUT_FORMATS` and `isOutputFormat`